serverless/cli_tools/installing-kn.adoc

```diff
@@ -18,20 +18,16 @@ endif::[]
 
 [IMPORTANT]
 ====
-If you try to use an older version of the Knative `kn` CLI with a newer {ServerlessProductName} release, the API is not found and an error occurs.
+If you try to use an older version of the Knative (`kn`) CLI with a newer {ServerlessProductName} release, the API is not found and an error occurs.
 
-For example, if you use the 1.16.0 release of the `kn` CLI, which uses version 0.22.0, with the 1.17.0 {ServerlessProductName} release, which uses the 0.23.0 versions of the Knative Serving and Knative Eventing APIs, the CLI does not work because it continues to look for the outdated 0.22.0 API versions.
+For example, if you use the 1.16.0 release of the Knative (`kn`) CLI, which uses version 0.22.0, with the 1.17.0 {ServerlessProductName} release, which uses the 0.23.0 versions of the Knative Serving and Knative Eventing APIs, the CLI does not work because it continues to look for the outdated 0.22.0 API versions.
```

Comment:
For minimalism purposes, can this be shortened? Can we simplify it to say:

'For example, if you use the 1.16.0 release of the Knative (`kn`) CLI with the 1.17.0 {ServerlessProductName} release, the CLI does not work because it continues to look for the outdated 0.22.0 API versions.`

Basically, do users _need_ to know what the Knative CLI uses and what the Serverless release uses? Right now, this warning is long and therefore kind of hard to read. 

However, if users need to know all this context, then keep it :)

Reply:
@abrennan89 I agree with Jakes suggestion, it's much clearer. Ok to rephrase?

Reply:
I think let's leave it as it is.

> 'For example, if you use the 1.16.0 release of the Knative (kn) CLI with the 1.17.0 {ServerlessProductName} release, the CLI does not work because it continues to look for the outdated 0.22.0 API versions.`

This doesn't really provide the whole context because it's not clear that 1.17.0 uses the 0.23.0 APIs, so a user might wonder what the relationship between the CLI and API versions is if it's not explained. It also removes the context about which APIs we're even referring to. I think for anything that majorly changes technical content like this, it's best to consult serverless QE first, rather than changing it in peer reviews after their reviews are done.

I would say the versions could maybe be updated to something more current, but that's probably out of scope for this release and should be addressed instead by the core serverless team. @SNiemann15 feel free to open a Jira for that.